src/Uno.UI/Helpers/WinUI/SharedHelpers.cs

```diff
@@ -31,13 +31,13 @@ namespace Uno.UI.Helpers.WinUI
 {
 	internal class SharedHelpers
 	{
-#pragma warning disable CS0414
-		private static bool s_isOnXboxInitialized = false;
-		private static bool s_isOnXbox = false;
-#pragma warning restore CS0414
+#if HAS_UNO && !(NET461 || __NETSTD_REFERENCE__)
```

Comment:
Why is this needed ?

Reply:
All usages are under the same condition (see method `IsOnXbox` in the same file).
If this is not guarded with this condition, we'll get CS0169 warning. I can add `#pragma warning disable CS0169`, but thought it would be better to be error-free and make it more clear for human reading the code that this fields are used under specific conditions.

Reply:
Makes sense.